go.mod

```diff
@@ -2,4 +2,7 @@ module k8s.io/klog/v2
 
 go 1.12
 
-require github.com/go-logr/logr v0.1.0
+require (
+	github.com/go-logr/logr v0.1.0
+	github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
```

Comment:
where is the glog dep coming from?  We should not need that

Reply:
go.mod from master is not updated. if you run `go mod tidy` from master, it also adds it

Reply:
That's super annoying.  @dims - I see it is just an example file.  We should Just turn that into a doc or something, get rid of the dep!

Reply:
I removed the go.mod changes and created a new issue for that https://github.com/kubernetes/klog/issues/129